dom-manipulation/main.js

```diff
@@ -0,0 +1,49 @@
+var hotButton = document.querySelector('.hot-button');
+var clickCount = document.querySelector('.click-count');
+var btn = document.querySelector('#btn');
+var clickCounter = 0;
+
+hotButton.addEventListener('click', function () {
+  clickCounter++;
+  clickCount.textContent = 'Clicks:' + ' ' + clickCounter;
+  if (clickCounter < 4) {
+
+    if (!btn.classList.contains('cold')) {
+      btn.classList.add('cold');
+    }
```

Comment:
I believe that this conditional may be unnecessary. You will need to try this out, but I believe that the `add` method of `classList` will not add duplicate classes. If you try to add a class that is already on the button, nothing happens. Try removing this and see if it works. If it does, you should remove the other conditionals as well to check if the classList already includes the next class.

Reply:
Hi Cody I tried removing the if statements and but once I remove them the button does not change colors as it does now it just goes to white.

Reply:
Actually I think I fixed it.